### DISCUSSION TITLE
[Question] How to add a vertical line

### DISCUSSION THREAD

**Discussion by TheTimKiely** [naver/billboard.js#3322] [General]
Hi,
Sorry if I'm missing something obvious.  I have a time series line chart and I'd like to add a vertical line to mark certain points in time.
Is this supported?  I can't find any reference to this functionality.
Thanks!

**Comment by netil** [comment]
Hi @TheTimKiely, I'm not sure but some vertical line like below example?
- https://naver.github.io/billboard.js/demo/#Grid.OptionalXGridLines
